dreem/inference/batch_tracker.py

```
@@ -0,0 +1,774 @@
+                logger.debug("New Video! Resetting Track Queue.")
+                self.track_queue.end_tracks()
+
+            """
+            Initialize tracks on first frame where detections appear. This is the first frame of the first batch
+            """
+            if len(self.track_queue) == 0:
+                if frame_to_track.has_instances():
+
+                    logger.debug(
+                        f"Initializing track on clip ind {batch_idx} frame {frame_to_track.frame_id.item()}"
+                    )
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)